Buildscripts/Flashing/merge.sh

```
@@ -0,0 +1,50 @@
+  else 
+    esptoolPath=esptool.py
+  fi
+else
+  esptoolPath=esptool
+fi
+
+chip=${1:-esp32s3}
+
+# Take the flash_arg file contents and join each line in the file into a single line
+flash_args=`tr '\n' ' ' < Binaries/flash_args`
+cd Binaries
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)